comment_scan/chatter.rb

```diff
@@ -57,33 +58,48 @@ def add_fall_through_reply_action(action, args_to_pass=nil)
     end
 
     def mention_received(room_id, message)
+        #Grab/create/update chat user
+        chat_user = ChatUser.find_or_create_by(user_id: message.hash['user_id'])
+        chat_user.update(name: message.hash['user_name'])
+
         @mention_actions.each do |action, payload|
```

Comment:
I think you want `#any?`

Reply:
Ooh you're right, that *should* short circuit. That's clever!